src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -211,6 +211,20 @@ function validate_boolean_string_value(value) {
     return false;
 }
 
+/**
+ * validate_no_leading_whitespace validates that a given value is not empty or whitespace-only
+ * @param {string} option
+ * @param {string} value
+ * @throws {ManageCLIError}
+ */
+function validate_no_leading_whitespace(option, value) {
+    // ensure values are not empty or whitespace only strings
+    if (value.trim() === '') {
+        const details = `flag ${option} cannot have an empty value or contain only whitespace. Please provide a valid value for ${option}`;
```

Comment:
I think the message might be confusing as the user did not passed an empty value or only whitespace characters (in you example: `--access_key= 12345678912345678ABC --secret_key= a234567891234567891212345678912345678912`

Reply:
Updated, Thanks